dev/devicelab/README.md

```diff
@@ -1,112 +1,51 @@
-# Flutter devicelab
+# Flutter DeviceLab
 
-"Devicelab" (a.k.a. [Cocoon](https://github.com/flutter/cocoon)) is a physical
-lab that tests Flutter on real Android and iOS devices.
+DeviceLab is a physical lab that tests Flutter on real devices.
 
-This package contains the code for the test framework and the tests. More generally
+This package contains the code for the test framework and tests. More generally
 the tests are referred to as "tasks" in the API, but since we primarily use it
 for testing, this document refers to them as "tests".
 
 Current statuses for the devicelab are available at
-https://flutter-dashboard.appspot.com.
+https://flutter-dashboard.appspot.com. See [dashboard user guide](https://github.com/flutter/cocoon/blob/master/app_flutter/USER_GUIDE.md)
+for information on using the dashboards.
 
-# Dashboards
+## How the DeviceLab runs tasks
 
-## Build dashboard
+The DeviceLab devices continuously ask Flutter's continuous integration system
+[Cocoon](https://github.com/flutter/cocoon) for tasks to run. When Cocoon has a
+task that is suitable for the device (e.g. Android test), it reserves that
```

Comment:
Nit: A link to https://github.com/flutter/flutter/blob/master/dev/devicelab/manifest.yaml#L21?

Reply:
Thanks, I pointed to the file without line number to prevent breakages there.